subprojects/docs/src/docs/userguide/build_environment.adoc

```diff
@@ -161,7 +161,7 @@ You should check for existence of optional project properties before you access
 
 You can adjust JVM options for Gradle in the following ways:
 
-The `org.gradle.jvmargs` Gradle property controls the VM running the build. It defaults to `-Xmx512m`
+The `org.gradle.jvmargs` Gradle property controls the VM running the build. It defaults to `-Xmx512m "-XX:MaxMetaspaceSize=256m"`
```

Comment:
```suggestion
The `org.gradle.jvmargs` Gradle property controls the VM running the build. It defaults to `-Xmx512m "-XX:MaxMetaspaceSize=320m"`
```

Reply:
This was just to find out why some tests still failed. We still leak a small amount of memory. I'll have to increase it for the user guide tests.